ang/civicase/CaseDetails--tabs--summary.html

```diff
@@ -44,6 +44,78 @@
     </div>
   </div>
   <!-- End - Subject and QuickLinks -->
+  <div class="civicase__summary-tab-row clearfix">
+    <div class="civicase__summary-tab-tile col-lg-6">
```

Comment:
Why do we need `civicase__summary-tab-tile`? instead you can add extra padding in `civicase__summary-tab-row` itself.

Reply:
We need `civicase__summary-tab-tile` to add padding to the left and right of two tiles. Please let me know if we need to have a call for this.